ProxyAppsSuite/tests/examini.yaml

```
@@ -0,0 +1,118 @@
+examini:
+   summary: Research vehicle for particule codes in Molecular Dynamics which allows modular independent investigation of different aspects such as force calculation, communication, neighbor list construction and binning.
+   
+   subtitle: "{{input.test_id}}"
+   variables:
+     input:
+        - { test_id: "test1", file: "in.lj", comm_type: "MPI", threads: '12', devices: '2' }    
+
+   permute_on: input
+   run:
+     modules: [ gcc, cmake, openmpi/2.1.2 ]
+     env:
+        OMP_PLACES: 'THREADS'
+        OMP_PROC_BIND: 'SPREAD'
+ 
+     cmds:
+         - cd ExaMiniMD/src
+         - srun ./ExaMiniMD  -il ../input/{{input.file}} --comm-type "{{input.comm_type}}" --kokkos-threads={{input.threads}} --kokkos-num-devices={{input.devices}}  &> {{input.test_id}}.out 
+         - cd ../../
+         - mv ExaMiniMD/src/{{input.test_id}}.out {{input.test_id}}.out
+         - cat {{input.test_id}}.out
+  
+   scheduler: slurm 
+   slurm:
+      num_nodes: 1
+      tasks_per_node: 2
+   
+   build:
+       modules: [ gcc, cmake, openmpi/2.1.2 ]
+       source_path: ExaMini
+       env:
+          CC: mpicc
+       cmds:
+           # Building KOKKOS
+           # Assumes kokkos was cloned into a directory at the same level as ExaMiniMD
+           # git clone https://github.com/kokkos/kokkos ~/kokkos
```

I wouldn't leave this commented out. When writing a test, assume users have nothing in their test_src directories already. 
```suggestion
           - git clone https://github.com/kokkos/kokkos
```